src/graph_notebook/seed/queries/propertygraph/identity/01_persistent_nodes_gremlin.txt

```diff
@@ -0,0 +1,4 @@
701376").addV("persistentId").property(id,"89be1aaaba157f3522bd3985be6d5624cba79003").property("pid","89be1aaaba157f3522bd3985be6d5624cba79003").addV("persistentId").property(id,"18de39a4f39e4065e247d2be61fd996ffc7bb6a1").property("pid","18de39a4f39e4065e247d2be61fd996ffc7bb6a1").addV("persistentId").property(id,"422f37aa23e1aa9ccdc6515a1b040274a3d462ba").property("pid","422f37aa23e1aa9ccdc6515a1b040274a3d462ba").addV("persistentId").property(id,"fa071d8446e504223d45bd7ac3f7dfbdf2ad4949").property("pid","fa071d8446e504223d45bd7ac3f7dfbdf2ad4949").addV("persistentId").property(id,"e3143f68533e3bf8362da6aad214f9525733a6db").property("pid","e3143f68533e3bf8362da6aad214f9525733a6db").addV("persistentId").property(id,"a0db638c9d91b104679a85023cdeee6ab0303d8b").property("pid","a0db638c9d91b104679a85023cdeee6ab0303d8b").addV("persistentId").property(id,"07b5f59736b09a97a899f4810755e802c541761e").property("pid","07b5f59736b09a97a899f4810755e802c541761e").addV("persistentId").property(id,"ed801b7f4439c9e1a48c3195c2bead14e75df931").property("pid","ed801b7f4439c9e1a48c3195c2bead14e75df931").addV("persistentId").property(id,"58b0c860c7df2c80b5a67d32ace29430f86b9a70").property("pid","58b0c860c7df2c80b5a67d32ace29430f86b9a70").addV("persistentId").property(id,"64f33e481e61e97cdc26bbd2dc106876d903db4c").property("pid","64f33e481e61e97cdc26bbd2dc106876d903db4c").addV("persistentId").property(id,"846b81d7ef670d1c8a90b4e53441ce8688be0918").property("pid","846b81d7ef670d1c8a90b4e53441ce8688be0918").addV("persistentId").property(id,"7b86943917e4041b77d2f4fe8ba59bc68b90322b").property("pid","7b86943917e4041b77d2f4fe8ba59bc68b90322b").addV("persistentId").property(id,"f52e86d65009cb4cb095b633fb8f8297007420f7").property("pid","f52e86d65009cb4cb095b633fb8f8297007420f7").addV("persistentId").property(id,"380c44b896026385dfd13655eb527380995694d1").property("pid","380c44b896026385dfd13655eb527380995694d1").addV("persistentId").property(id,"6d9fcf1e367fe13bd7405ddb17483f425a8d6944").property("pid","6d9fcf1e367fe13bd7405ddb17483f425a8d6944").addV("persistentId").property(id,"f1a42fd42be8f88695e1a6fced77b10e01ab43d0").property("pid","f1a42fd42be8f88695e1a6fced77b10e01ab43d0").addV("persistentId").property(id,"5c3000a90eb38eeb34da252fd675aea17f915f5b").property("pid","5c3000a90eb38eeb34da252fd675aea17f915f5b").addV("persistentId").property(id,"8ecf79339d8df1e00dea903be00f6013821a70e6").property("pid","8ecf79339d8df1e00dea903be00f6013821a70e6").addV("persistentId").property(id,"7a976baa095047b5fcaa97e939fd723058b4c06a").property("pid","7a976baa095047b5fcaa97e939fd723058b4c06a").addV("persistentId").property(id,"c092d73a469471b9303e9d3c5b755f362d2106be").property("pid","c092d73a469471b9303e9d3c5b755f362d2106be").addV("persistentId").property(id,"73a5add082977b7d1588f51ab5e057907e6402a7").property("pid","73a5add082977b7d1588f51ab5e057907e6402a7").addV("persistentId").property(id,"6b12fd2aa1638468206b47c2d9d53e3cabd1afff").property("pid","6b12fd2aa1638468206b47c2d9d53e3cabd1afff")
```

Comment:
I don't think these files are supposed to be included in the package due to size

Reply:
Removed